public/modules/custom/helfi_rekry_content/src/Form/SettingsForm.php

```diff
@@ -81,6 +82,22 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#description' => $this->t('This page is displayed for anonymous users when a job listing is unpublished. Redirect to page that contains information about old and removed job listings.'),
     ];
 
+    $language = \Drupal::languageManager()->getCurrentLanguage(LanguageInterface::TYPE_CONTENT)->getId();
```

Comment:
Instead of implementing a new way to save translated text, it's probably a better to use the built-in way to translate configs. So basically add the fields to `helfi_rekry_content.schema.yml` file, add the `helfi_rekry_content.config_translation.yml` file, and also add the `helfi_rekry_content.links.task.yml` file to show the link tab to the translation forms.

Reply:
Now implemented as suggested, thanks!